text/0005-architecture-reviews.md

```diff
@@ -0,0 +1,135 @@
+
+The audience of a Roadmap Review is those "up the chain" of decision makers and the audience of a Design Review is those running “down the chain” of implementers.
+
+A Roadmap Review should focus on:
+
+* The problem space
+* Large forces impacting Mozilla’s choices in the space
+* Competitive analysis
+* Opportunity cost
+
+The key outcome is senior management investing in the problem space and building consensus around a shared reality.
+
+A Design Review should focus on:
+
+* The solution space
+* Technical roadmaps
+* Milestones and timelines
+* Resource allocation
+
+The key outcome is middle management and individual engineers understanding the value proposition and the path to success.
+
+In [RACI](https://en.wikipedia.org/wiki/Responsibility_assignment_matrix) terms. The team is Responsible, the chair is accountable, other senior engineers or managers are Consulted and various others in the organization are Informed.
+
+### Structure and inputs
+
+The second two considerations shape how the review process actually happens.
+
+The chair and the team frame the question or questions at the start of the process to ensure agreement on what should be tested. The chair is accountable for answering the question or questions following the review process.
+
+In order to agree the question(s), the chair and the team/technical lead or principles will generally have an initial meeting. This meeting should be as small possible and is intended to:
+
+* establish vocabulary
+* help the team "argue the right level" for the review
+* identify stakeholders and domain experts — those Consulted, per RACI — in the organization
+
+Subsequently, the team prepares a review packet detailing the problem space and their approach.
+
+The goal is to have the chair (representing the responsible decision makers) and the team establish some shared reality informally, and then to have the team document this and explain how their work addresses the decision maker’s question within that reality.
+
+The chair then evaluates and refines the team’s document with the assistance of domain experts and the identified stakeholders. Hopefully the consulted stakeholders concur with the framing of the problem and are aligned behind the architectural approach, but perhaps they are not. The chair collects such concerns and presents them. This should be a collaborative process whereby the architectural review functions to document a problem and a proposed solution, in which case the chair can solicit response from the team.
+
+Hopefully, involving the team and the chair informally and subsequently sharing ownership of a document will seed a collaborative process. A particular benefit is avoiding drift between the proposal and the focus of the review: that is, a "successful review" that results in a “go” or “no go” from responsible decision makers in which the review doesn't actually evaluate what the team proposes.
```

Comment:
`Hopefully` doesn't engender confidence in the outcome of this step. Make it more definite.